src/sdk/namespace_s3.js

```diff
@@ -184,7 +184,7 @@ class NamespaceS3 {
                     req.removeListener('httpError', AWS.EventListeners.Core.HTTP_ERROR);
                     let count = 1;
                     const count_stream = stream_utils.get_tap_stream(data => {
-                        stats_collector.instance(this.rpc_client).update_namespace_read_stats({
+                        stats_collector.instance(this.rpc_client).update_hub_read_stats({
```

Comment:
I think the previous name made more sense. 

Also you seem to change only namespace_s3 but not namespace_blob...

But first I'm thinking - can we separate the hub metric update and call it from namespace_cache?

Reply:
I just wanted to differentiate more clearly between hub-related stats and cache-related stats. Is there a name that you think would make sense and would still have that differentiation?

Re the namespace_blob change, I forgot to change it

What do you mean about separating the hub metric update?